### PR TITLE
Vagrant files

### PR DESCRIPTION
**Pull request by ctienshi**
## Automation tests
 -N/A

## Security checks
N/A

## Related PRs
N/A

## Migrations (if applicable)
N/A

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    cpu: 1
+    provisioner_script: worker/provisioner/product_provisioner.sh
+  -
+    hostname: dashboard
+    box: wso2sp
+    ip: 172.28.128.6
+    ram: 2048
+    cpu: 1
+    provisioner_script: dashboard/provisioner/product_provisioner.sh
+
```
Comment (VimukthiPerera) [inline config.yaml]: unnecessary space

File: config.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    ram: 2048
+    cpu: 1
+    provisioner_script: worker/provisioner/product_provisioner.sh
+  -
+    hostname: dashboard
+    box: wso2sp
+    ip: 172.28.128.6
+    ram: 2048
+    cpu: 1
+    provisioner_script: dashboard/provisioner/product_provisioner.sh
```
Comment (DilanUA) [inline config.yaml]: Enter new line here

File: dashboard/provisioner/product_provisioner.sh [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+# tail the WSO2 product server startup logs until the server startup confirmation is logged
+tail -f ${WORKING_DIRECTORY}/${WSO2_SERVER}-${WSO2_SERVER_VERSION}/wso2/dashboard/logs/carbon.log | while read LOG_LINE
+do
+  # echo each log line
+  echo "${LOG_LINE}"
+  # once the log line with WSO2 Carbon server start confirmation was logged, kill the started tail process
+  [[ "${LOG_LINE}" == *"WSO2 Carbon started"* ]] && pkill tail
+done
+
```
Comment (DilanUA) [inline dashboard/provisioner/product_provisioner.sh]: Additional new line

File: manager/provisioner/product_provisioner.sh [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+fi
+
+export JAVA_HOME
+
+# start the WSO2 product pack as a background service
+echo "Starting ${WSO2_SERVER}-${WSO2_SERVER_VERSION}..."
+nohup ${WORKING_DIRECTORY}/${WSO2_SERVER}-${WSO2_SERVER_VERSION}/wso2/manager/bin/carbon.sh &
+
+sleep 10
+
```
Comment (DilanUA) [inline manager/provisioner/product_provisioner.sh]: Additional new line

**Review by DilanUA — APPROVED** [review]

**Review by msmshariq — APPROVED** [review]

**Pull request merged**